### DISCUSSION TITLE
Permission to submit flatpak to flathub

### DISCUSSION THREAD

**Discussion by Lunarequest** [strawberrymusicplayer/strawberry#800] [General]
I've made a flatpak here, https://github.com/Lunarequest/stawberry-flatpak I was wondering if I  could have permission to submit it to flathub

**Comment by jonaski** [comment]
I'm moving this to discussions since it's not an bug/issue.

**Reply by Lunarequest** [reply]
So would it be fine for me to submit this to flathub?

**Reply by jonaski** [reply]
I was working on the answer :)
I guess so, but you need to make sure both alsa and pulseaudio devices can be listed in the backend settings, and that both alsa and pulseadio can be used.
Also a good idea to test a couple of udisks2 devices, just a memory stick or SD card should be sufficient, check that they can be mounted, files listed and umounted in Strawberry under the devices tab.
It might also be a good idea to compile with Qt 6 if that's available.

**Reply by Lunarequest** [reply]
Qt 6 would have to be compiled from source, currently the udisk permissions aren't set, I'll add them. Both alsa and pulseaudio work as intended.

**Reply by Lunarequest** [reply]
how does the app interact with udisks. does it directly interact with the udisks dbus interface?

**Reply by jonaski** [reply]
Yes

**Reply by Lunarequest** [reply]
I can confirm it works. I'll cleanup and up submit the flatpak in the morning

**Comment by Lunarequest** [comment]
Pr is here https://github.com/flathub/flathub/pull/2590 for anyone wondering

**Reply by Lunarequest** [reply]
It is now on flathub https://flathub.org/apps/details/org.strawberrymusicplayer.strawberry (it will appear here shortly)

**Reply by bhankas** [reply]
Does this mean this discussion can now be closed (or equivalent, not really familiar with this feature)?

**Reply by Lunarequest** [reply]
Yes it can but i do not believe there is a way to close a discussion
